h/search/persistence.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+"""Code for persisting (indexing) annotations to Elasticsearch."""
```

Comment:
With this being a new file I was able to format it with <https://github.com/ambv/black> :)